### PR TITLE
chore(build): add more enterprise build support

### PR DESCRIPTION
**Pull request by turtleDeng**
Fixes <issue-number>

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/emqx/emqx/blob/master/CONTRIBUTING.md.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] In case of non-backward compatible changes, reviewer should check this item as a write-off, and add details in **Backward Compatibility** section

## More information

### CODE REVIEW

File: bin/emqx [hunk 1]
```diff
@@ -209,7 +209,7 @@ generate_config() {
         # the vm, we need to pass it in twice.
         CONFIG_ARGS=" -config $RUNNER_ETC_DIR/app.config -args_file $RUNNER_ETC_DIR/vm.args -vm_args $RUNNER_ETC_DIR/vm.args "
     else
-        CONFIG_ARGS=$("$ERTS_PATH"/escript "$RUNNER_ROOT_DIR"/bin/cuttlefish -i "$REL_DIR"/emqx.schema -c "$RUNNER_ETC_DIR"/emqx.conf -d "$RUNNER_DATA_DIR"/configs generate)
+        CONFIG_ARGS=$("$ERTS_PATH"/escript "$RUNNER_ROOT_DIR"/bin/cuttlefish -i "$REL_DIR"/emqx.schema $EMQX_LICENSE -c "$RUNNER_ETC_DIR"/emqx.conf -d "$RUNNER_DATA_DIR"/configs generate)
```
Comment (zmstone) [inline bin/emqx:212]:
```suggestion
        EMQX_LICENSE_CONF_OPTION=""
        if [ "${EMQX_LICENSE_CONF:-}" != "" ]; then
            EMQX_LICENSE_CONF_OPTION="-i ${EMQX_LICENSE_CONF}"
        fi
        CONFIG_ARGS=$("$ERTS_PATH"/escript "$RUNNER_ROOT_DIR"/bin/cuttlefish -i "$REL_DIR"/emqx.schema $EMQX_LICENSE_CONF_OPTION -c "$RUNNER_ETC_DIR"/emqx.conf -d "$RUNNER_DATA_DIR"/configs generate)
```

File: data/emqx_vars [hunk 2]
```diff
@@ -16,6 +16,7 @@ RUNNER_USER="{{ runner_user }}"
 EMQX_DISCR="{{ emqx_description }}"
 LIB_EKKA_DIR="${RUNNER_LIB_DIR}/ekka-$(grep ekka "${RUNNER_ROOT_DIR}/releases/RELEASES" | awk -F '\"' '{print $2}')"
 
+EMQX_LICENSE=''
```
Comment (zmstone) [inline data/emqx_vars:19]: EMQX_LICENSE_CONF=''

**Review by terry-xiaoyu — DISMISSED** [review]

File: rebar.config.erl [hunk 3]
```diff
@@ -225,28 +225,31 @@ relx_plugin_apps_extra() ->
...
+    , {template, "bin/emqx.cmd", "bin/emqx.cmd"}
+    , {template, "bin/emqx_ctl.cmd", "bin/emqx_ctl.cmd"}
+    , {copy, "bin/nodetool", "bin/nodetool"}
+    , {copy, "bin/nodetool", "bin/nodetool-{{release_version}}"}
+    , {copy, "_build/default/lib/cuttlefish/cuttlefish", "bin/cuttlefish"}
+    , {copy, "_build/default/lib/cuttlefish/cuttlefish", "bin/cuttlefish-{{release_version}}"}
+    , {copy, "priv/emqx.schema", "releases/{{release_version}}/"}
+    ] ++ ee_etc_overlay() ++ etc_overlay(ReleaseType).
+
+ee_etc_overlay() ->
```
Comment (zmstone) [inline rebar.config.erl:251]: move it to the bottom of this file.

**Review by zmstone — APPROVED** [review]

**Comment by zmstone** [comment]
maybe rebase your commits.

**Review by terry-xiaoyu — APPROVED** [review]

**Pull request merged**